src/OrleansTelemetryConsumers.Counters/OrleansPerfCounterTelemetryConsumer.cs

```diff
@@ -10,61 +10,26 @@ namespace OrleansTelemetryConsumers.Counters
     /// <summary>
     /// Telemetry consumer that writes metrics to predefined performance counters.
     /// </summary>
-    public class OrleansPerfCounterTelemetryConsumer : IMetricTelemetryConsumer
+    internal class OrleansPerfCounterTelemetryConsumer : IMetricTelemetryConsumer
```

Comment:
Why was this made internal? wouldn't this mean that users cannot use programmatic config to add this telemetry consumer?